app/Resources/migrations/generateMigration.php

```diff
@@ -4074,6 +4074,13 @@ protected function getAddIndexes()
             'unique' => false
         );
 
+        $changes[] = array(
```

Comment:
Can you move this up a few lines to be sort of alphabetical by table?  Its the only way to make some kind of sense of this file.